### PR TITLE
Initial release of episode 18.1

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/6479

* **Server Mode**: renewal

* **Description of Pull Request**: 

- [x] Code review
- [ ] ~~English review~~ (later)

### CODE REVIEW

**Comment by eppc0330** [comment]
Nice

**Comment by idk-whoami** [comment]
Camille's health score was negative

script does not end

https://github.com/rathena/rathena/assets/60462533/73ff2ef0-5c2e-444a-92a9-e02054dccc96


ref https://youtu.be/Bfk7P17y02U?t=2973

**Comment by idk-whoami** [comment]
Instance: Villa of High Priest.

Capture
```
[Warning]: buildin_monster: Attempted to spawn non-existing monster class 20562
[Warning]: Script command 'monster' returned failure.
[Debug]: Source (NPC): adv_demigod_boss at 1@adv (1,1)
[Debug]: Source (NPC): adv_demigod_boss is located in: INSTANCING
```
when the last MVP spawns

**Comment by idk-whoami** [comment]
REPUTATION DB missing

Capture

```
[Error]: buildin_set_reputation_points: Unknown reputation type 5.
[Warning]: Script command 'add_reputation_points' returned failure.
[Debug]: Source (NPC): Ahir#ep18 at wolfvill (147,97)
[Debug]: Source (NPC): Ahir#ep18 is located in: npc/re/quests/quests_18_1.txt
[Error]: buildin_set_reputation_points: Unknown reputation type 5.
[Warning]: Script command 'get_reputation_points' returned failure.
[Debug]: Source (NPC): Ahir#ep18 at wolfvill (147,97)
[Debug]: Source (NPC): Ahir#ep18 is located in: npc/re/quests/quests_18_1.txt
[Error]: script:op_2: invalid data for operator C_GE
[Debug]: Data: C_ARG
[Debug]: Data: number value=1000
[Debug]: Source (NPC): Ahir#ep18 at wolfvill (147,97)
[Debug]: Source (NPC): Ahir#ep18 is located in: npc/re/quests/quests_18_1.txt

File: npc/re/quests/quests_18_1.txt [hunk 1]
```diff
@@ -0,0 +1,12429 @@
...
+		mes "Are you saying this is the story that Shanina Grandma told me?";
+		mes "A story that her grandmother grew up listening to when she was young....";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "I will read this carefully and organize it.";
+		mes "Please continue to receive stories from Shanina Grandma.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "I hope you will come back tomorrow and dictate the next part of this story.";
+		mes "Thank you. This is your pay for today's work.";
```
[the remark this answers is not in the document]
Comment (Kuyaraikub) [reply npc/re/quests/quests_18_1.txt:7214]: And all quest when finish can't close dialog.

File: npc/re/quests/quests_18_1.txt [hunk 2]
```diff
@@ -0,0 +1,12429 @@
...
+		next;
+		mes "[Amira]";
+		mes "Anyway, I promised to tell you the story I heard from my late grandfather before we came to the Gray Wolf Forest.";
+		next;
+		mes "[Amira]";
+		mes "Are you ready to listen? Now, I'll speak slowly.";
+		mes "There are many more gods in the world than we know...";
+		completequest 16552;
+		if (isbegin_quest(16551) == 2 && isbegin_quest(16553) == 2) {
+			npctalk "I wrote down all three stories. Let's go b
```
[the remark this answers is not in the document]
Comment (Atemo) [reply npc/re/quests/quests_18_1.txt:8958]: Update your emulator.

**Pull request merged**